cmd/pinniped-proxy/src/main.rs

```diff
@@ -27,17 +27,18 @@ mod tls_config;
 #[tokio::main]
 async fn main() -> Result<()> {
     pretty_env_logger::formatted_timed_builder()
-    .format(|buf, record| {
-        writeln!(buf,
-            "{} [{}] - {}",
-            Local::now().format("%Y-%m-%dT%H:%M:%S%.6f"),
-            record.level(),
-            record.args()
-        )
-    })
-    .filter(None, LevelFilter::Info)
-    .init();
-    let opt = cli::Options::from_args();
+        .format(|buf, record| {
+            writeln!(
+                buf,
+                "{} [{}] - {}",
+                Local::now().format("%Y-%m-%dT%H:%M:%S%.6f"),
```

Comment:
I don't really recall right now if the rest of the log traces in the Kubeapps components are being emitted in local time or in UTC time. I mean, not for this PR, but I wanted to bring this up just in case.

Reply:
Should be the same thing (though is not guaranteed to be. Most images are built with UTC as the local TZ, but yep, it may be worth verifying.